### PR TITLE
Support the --destination flag on galasactl runs download

### PR DESCRIPTION
**Pull request by techcobweb**
Signed-off-by: Mike Cobbett <77053+techcobweb@users.noreply.github.com>

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

File: pkg/runs/runsDownload.go [hunk 1]
```diff
@@ -133,11 +148,25 @@ func nameDownloadFolder(run galasaapi.Run, runName string, timeService utils.Tim
...
+) error {
+
+	runId := run.GetRunId()
+
+	// We want to base the directory we download to on the destination folder.
+	// If the destination folder is "." (current folder/relative)
+	// then ignore, as prefixing with a "." just adds noise when we log and
+	// print out the path.
+	if runDownloadTargetFolder != "." {
+		directoryName = runDownloadTargetFolder + "/" + directoryName
```
Comment (eamansour) [inline pkg/runs/runsDownload.go:165]: Might be worth using `filepath.Join` to join path elements with an OS-specific separator?

**Review by eamansour — APPROVED** [review]
Looks good - just have one minor suggested change below, but happy to approve.

**Review by AaronJhaj — APPROVED** [review]
I like the new console message logic when creating a folder

File: pkg/runs/runsDownload.go (hunk 1 above)
Comment (techcobweb) [reply pkg/runs/runsDownload.go]: Yep. Used that. Good suggestion. Thanks.

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**